src/SFA.DAS.RoATPService.Application.Api/Controllers/LookupDataController.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace SFA.DAS.RoATPService.Application.Api.Controllers
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using MediatR;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Logging;
+    using RoATPService.Api.Types.Models;
+    using SFA.DAS.RoATPService.Domain;
+
+    [Authorize(Roles = "RoATPServiceInternalAPI")]
+    [Route("api/v1/lookupData")]
+    [ApiController]
+    public class LookupDataController : Controller
+    {
+        private ILogger<LookupDataController> _logger;
+
+        private IMediator _mediator;
+
+        public LookupDataController(ILogger<LookupDataController> logger, IMediator mediator)
+        {
+            _logger = logger;
+            _mediator = mediator;
+        }
+
+        [Route("providerTypes")]
+        public async Task<IActionResult> ProviderTypes()
+        {
+            var request = new GetProviderTypesRequest();
+
+            IEnumerable<ProviderType> providerTypes = await _mediator.Send(request);
+
+            return Ok(providerTypes);
+        }
+
+        [Route("organisationTypes")]
+        public async Task<IActionResult> OrganisationTypes(int providerTypeId)
+        {
+            var request = new GetOrganisationTypesRequest { ProviderTypeId = providerTypeId };
+
+            IEnumerable<OrganisationType> organisationTypes = await _mediator.Send(request);
```

Comment:
shouldnt need strong typing?  Also you could fold this into   return Ok(await _mediator.Send(request) maybe?

Reply:
Will simply the code, good point, the contract may change in future